src/stream/pipeline/v4l_pipeline.rs

```diff
@@ -74,9 +74,43 @@ impl V4lPipeline {
                 tee_name = PIPELINE_TEE_NAME
             )
             }
+            VideoEncodeType::YUYV => {
+                format!(concat!(
+                "v4l2src device={device} do-timestamp=false",
+                " ! videoconvert",
+                " ! capsfilter name={filter_name} caps=video/x-raw,format=UYUV,sampling=YCbCr-4:2:2,width={width},height={height},framerate={interval_denominator}/{interval_numerator}",
+                " ! rtpvrawpay pt=96",
+                " ! tee name={tee_name} allow-not-linked=true"
+            ),
+            device = device,
+            width = width,
+            height = height,
+            interval_denominator = interval_denominator,
+            interval_numerator = interval_numerator,
+            filter_name = PIPELINE_FILTER_NAME,
+            tee_name = PIPELINE_TEE_NAME
+            )
+            }
```

Comment:
the code format looks weird

Reply:
Yeah, I didn't like it either, but couldn't be different :(

Reply:
cargo fmt was just failing to automatically get it right but fixing it by hand works :) Thanks!